tinyblog/articles/2016-07-01-registration.md

```diff
@@ -0,0 +1,72 @@
+# 2016 Registration
+
+> Published on 1st Jul 2016 by Ola Sitarska
+
+Every year tickets to Django: Under The Hood are selling out in a matter of minutes.
+This year we're trying something different to make the distribution of tickets
+a little bit more fair and less frustrating for everyone.
+
+Read on to learn about the important changes we're making this year!
+
+---
+
+#### The Short Version
+
+1. __On Tuesday, July 12, we're going to open registration.__
+You'll have two weeks to submit your e-mail address for the lottery, until July 19th at noon CET.
```

Comment:
I find it useful to like times to `time.is`: http://time.is/1200_19_July_2016_in_Amsterdam